src/base/bittorrent/torrenthandle.cpp

```diff
@@ -915,26 +915,29 @@ int TorrentHandle::leechsCount() const
 
 int TorrentHandle::totalSeedsCount() const
 {
-    return m_nativeStatus.list_seeds;
+    return (m_nativeStatus.num_complete > 0) ? m_nativeStatus.num_complete : m_nativeStatus.list_seeds;
```

Comment:
>What if know peers that the tracker doesn't (eg from DHT, LSD)?

Then those peers won't be counted in.

>So you opted to use the tracker scrape info as more reliable?

That's what most users expect and the code is from (obsolete) libtorrent example code.